packages/fxa-auth-server/lib/senders/email.js

```diff
@@ -1934,7 +1932,7 @@ module.exports = function (log, config) {
       return;
     }
 
-    const query = { email, productName, token, code, redirectUrl };
+    const query = { email, productName, token, code, product_id: productId };
```

Comment:
nit: Looks like mix of camel and snake case. I think I introduced productName so wouldn't mind if converted to snake case.